js/deribit.js

```diff
@@ -1790,16 +1793,21 @@ module.exports = class deribit extends Exchange {
         for (let i = 0; i < positions.length; i++) {
             result.push (this.parsePosition (positions[i]));
         }
-        // todo unify parsePositions
         return result;
     }
 
     async fetchPositions (symbols = undefined, params = {}) {
         await this.loadMarkets ();
-        const code = this.codeFromOptions ('fetchPositions', params);
+        let code = undefined;
+        if (symbols === undefined) {
+            code = this.codeFromOptions ('fetchPositions', params);
+        } else {
+            code = this.safeSymbol (symbols);
```

Comment:
In this case we should not use safeSymbol – it is only used for recognizing and parsing exchange-specific symbols to the unified CCXT symbols on the response-parsing stage, but not on the request-sending stage.
We should replace line 1805 in this case with something like:

```JavaScript
} else if (!Array.isArray (symbols)) {
    code = symbols;
}
```